src/main/scala/com/gu/subscriptions/cas/model/ResponseComparer.scala

```diff
@@ -0,0 +1,25 @@
+package com.gu.subscriptions.cas.model
+
+import com.typesafe.scalalogging.LazyLogging
+import spray.http.HttpResponse
+
+object ResponseComparer extends LazyLogging {
+  def compare(response: HttpResponse, other: HttpResponse): Unit = {
+    val sanitize = (in:String) => in.replaceAll(" ", "").replaceAll("\n", "")
+    val responseBody: String = response.entity.asString
+    val otherResponseBody: String = other.entity.asString
+
+    def isSame = {
+      val hasSameStatus = response.status == other.status
+      val hasSameBody = sanitize(responseBody) == sanitize(otherResponseBody)
+      hasSameBody && hasSameStatus
+    }
+    if (!isSame) {
+      logger.error("Legacy apps returned different Responses")
```

Comment:
I'm guessing this line will activate a lot once valid Call Credit response are being returned by the new infrastructure!